Octokit/Helpers/HttpClientExtensions.cs

```diff
@@ -7,16 +7,12 @@ namespace Octokit
 {
     public static class HttpClientExtensions
     {
-        /// <summary>
-        /// OBSOLETE
-        /// </summary>
-        [Obsolete("This will be removed in a future release")]
-        public static Task<IResponse<T>> Send<T>(this IHttpClient httpClient, IRequest request)
+        public static Task<IResponse> Send(this IHttpClient httpClient, IRequest request)
```

Comment:
We can always add the extension method to the GHfW side and remove it from here. I don't feel strongly either way other than leaning towards being lazy. :stuck_out_tongue: